src/labkey/Domain.ts

```diff
@@ -387,6 +390,7 @@ export function save(config: SaveDomainOptions): XMLHttpRequest {
                   containerPath: arguments[5],
                   includeWarnings: arguments[6],
                   options: arguments[7],
+                  auditUserComment: arguments[8],
```

Comment:
nit: This isn't necessary. This interface is only intended to be backwards compatible with the original arguments list.